.test-infra/jenkins/job_LoadTests_SideInput_Flink_Go.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonTestProperties
+import CommonJobProperties as commonJobProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+import InfluxDBCredentialsHelper
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def fromTemplate = { mode, name, id, testSpecificOptions ->
+  [
+    title          : "SideInput Go Load test: ${name}",
+    test           : 'sideinput',
+    runner         : CommonTestProperties.Runner.FLINK,
+    pipelineOptions: [
+      job_name           : "load-tests-go-flink-${mode}-sideinput-${id}-${now}",
+      influx_namespace   : 'flink',
+      influx_measurement : "go_${mode}_sideinput_${id}_${now}",
```

Comment:
Why `${now}`? We would quickly end up with tons of different measurements in the database with only one element.

Reply:
Thank you, in the last commit I removed  ${now} from influx_measurement.